python/nano/src/bigdl/nano/deps/neural_compressor/pytorch/quantized_model.py

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from pathlib import Path
+import yaml
+from bigdl.nano.utils.inference.pytorch.model import AcceleratedLightningModule
+from neural_compressor.utils.pytorch import load
+from neural_compressor.model.model import PyTorchModel
+
+
+class PytorchQuantizedModel(AcceleratedLightningModule):
+    def __init__(self, model):
+        super().__init__(model.model)
+        self.quantized = model
+
+    def load_state_dict(self, state_dict):
+        self.quantized.model.load_state_dict(state_dict)
+
+    def state_dict(self):
+        return self.quantized.model.state_dict()
```

Comment:
Why we need these 2 methods(`load_state_dict`, `state_dict`)?

Reply:
If a pytorch user happen to use these two function, then they will get error.. A `PytorchQuantizedModel` in fact is a `nn.Module` too.